package.json

```diff
@@ -42,6 +42,7 @@
     "test": "npm run test:unit && npm run test:e2e:smoke",
     "test:unit": "npm run test:main && npm run test:preload && npm run test:preload-docker-extension && npm run test:renderer && npm run test:tools && npm run test:extensions",
     "test:e2e:smoke": "cross-env NODE_ENV=development MODE=development DEBUG=pw:browser npm run build && xvfb-maybe vitest run tests/src/e2e-smoke.spec.ts",
+    "test:e2e:pull-image": "cross-env NODE_ENV=development MODE=development DEBUG=pw:browser npm run build && xvfb-maybe vitest run tests/src/e2e-pull-image.spec.ts",
```

Comment:
I thought that all e2e were using a single task

like 'test:e2e:smoke' run all '*-smoke.spec.ts' and "test:e2e" run all e2e tests

if you want only "pull image tests" you can run a custom vitest command

Reply:
Sounds good. Except, I do not think that on GHA windows/mac pull image will work (did not test yet), since there is no podman available. Docker is though, but I am not sure. So I wanted to have `*-smoke.spec.ts` tests to be (possibly) run on all platforms on GHA. And then one distinguished npm task: `test:e2e:full` (or without `full` suffix) that will be run on linux that has podman available. 
I would propose this for smoke npm task:
`"test:e2e:smoke": "cross-env NODE_ENV=development MODE=development DEBUG=pw:browser npm run build && xvfb-maybe vitest run tests/src/*smoke.spec.ts --threads false",` - renaming `e2e-smoke.spec.ts` to `welcome-page-smoke.spec.ts`
And this for full:
`"test:e2e": "cross-env NODE_ENV=development MODE=development DEBUG=pw:browser npm run build && xvfb-maybe vitest run tests --threads false",`
And to rename the test `e2e-pull-image.spec.ts` to `pull-image.spec.ts`

@benoitf  @xbabalov wdyt?

Reply:
> Sounds good. Except, I do not think that on GHA windows/mac pull image will work (did not test yet)

yes but we launch smoke tests only on Linux:
https://github.com/containers/podman-desktop/blob/main/.github/workflows/pr-check.yaml#L218-L220

Reply:
about the renaming I'm fine

And it would be nice if everything that can run on Linux/Github Action could be launched on each PR check as well

Reply:
For now, true. 
So let's rename pull image test file to `pull-image-smoke.spec.ts` and let it be run as part of smoke suite with npm task: `"test:e2e:smoke": "cross-env NODE_ENV=development MODE=development DEBUG=pw:browser npm run build && xvfb-maybe vitest run tests/src/*smoke.spec.ts --threads false"`...